### PR TITLE
L6 Saw Bullet Tweaks

### PR DESCRIPTION
**Pull request by Mqiib**
# Document the changes in your pull request

Currently, AP rounds are literally just better versions of the normal rounds with obscenely high armor piercing. This tweaks them to be slightly less damaging and have slightly lower AP while still being essentially the best all-around bullets due to their (still high) AP. Regular L6 rounds have slightly higher chance to cause wounds. HP rounds are MUCH more effective now and can easily cause severe wounds on unarmored targets and even lightly-armored targets after a few hits to tear through the jumpsuit. They also have a much lower AP penalty against armor and shouldn't be the flaming trash they were before. Also removes their wound chance fall-off per tile because they were already bad enough.

# Wiki Documentation

Only number changes, which I don't think are explicitly on the wiki. 

# Changelog

:cl:  

tweak: Normal L6 round wound_bonus -50 -> -40
tweak: AP L6 round damage 40 -> 35
tweak: AP L6 round wound_bonus 0 -> -45
tweak: AP L6 round armour_penetration 75 -> 60
tweak: AP L6 round bare_wound_bonus 0 -> -10
tweak: HP L6 round armour_penetration -60 -> -35
tweak: HP L6 round wound_bonus -40 -> -35
tweak: HP L6 round bare_wound_bonus 30 -> 35
tweak: HP L6 round would_falloff_tile -8 -> 0

/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/bullets/lmg.dm [hunk 1]
```diff
@@ -27,22 +27,23 @@
...
+	bare_wound_bonus = -10			//flesh wont stop these very effectively but armor might make it tumble a bit before it enters
 
 /obj/item/projectile/bullet/mm712x82_hp
 	name = "7.12x82mm hollow-point bullet"
 	damage = 50
-	armour_penetration = -60
-	sharpness = SHARP_EDGED
-	wound_bonus = -40
-	bare_wound_bonus = 30
-	wound_falloff_tile = -8
```
Comment (adamsong) [inline code/modules/projectiles/projectile/bullets/lmg.dm:45]: Undocumented change

**Review by adamsong — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/bullets/lmg.dm (hunk 1 above)
Comment (adamsong) [reply code/modules/projectiles/projectile/bullets/lmg.dm:45]: (Specifically the wound falloff tile)

Comment (Mqiib) [reply code/modules/projectiles/projectile/bullets/lmg.dm:45]: Not in the changelog (which bibby yelled at me for making too long) but it is in the description.

**Review by adamsong — APPROVED** [review]
Numbers are changed as expected

**Comment by JamieD1** [comment]
@SomeguyManperson

**Pull request merged**